src/org/jitsi/impl/neomedia/rtp/RTPEncodingDesc.java

```diff
@@ -226,6 +226,7 @@ else if (tsDiff >= (-(1L << 30) & 0xFFFFFFFFL))
                 }
                 else if (snDiff < 2 || snDiff > (-3 & 0xFFFF))
                 {
+                    // FIXME we get that way too often.
```

Comment:
I re-read the code and you're right, forget what I said about `snDiff = RTPUtils.sequenceNumberDiff(max, min)`. I'll fix this, thanks @bgrozev.